presto-native-execution/scripts/setup-macos.sh

```diff
@@ -17,51 +17,24 @@ set -eufx -o pipefail
 source "$(dirname "${BASH_SOURCE}")/../velox/scripts/setup-macos.sh"
 
 MACOS_DEPS="${MACOS_DEPS} bison gperf libsodium"
-export FB_OS_VERSION=v2022.11.14.00
+export FB_OS_VERSION=v2023.12.04.00
```

Comment:
Are we hard-coding this in 2 places: Velox and here? Is this how we want this to be? Why can't we use the version specified in Velox?

Reply:
Strangely I don't see similar change in other setup-xxx.sh scripts. Why are they different?

Reply:
I believe Velox script gets invoked in a subshell whose environment might not be visible to the parent script here. So we need to define here as well.

Reply:
I will take a look if we can cache the Velox variable. But out of scope here.